tests/pipelines/eks/awscli-cl2-load-with-addons.yaml

```diff
@@ -28,34 +34,119 @@ spec:
     taskRef:
       kind: Task
       name:  slack-notification
+  - name: create-cluster-service-role
+    params:
+    - name: stack-name
+      value: $(params.cluster-name)-service-role
+    - name: role-cfn-url
+      value: $(params.service-role-cfn-url)s
```

Comment:
`s` at the end? typo?

Reply:
Unintentional change..I guess it got in by mistake.

Reply:
removed it.